test/core/security/grpc_tls_crl_provider_test.cc

```diff
@@ -88,9 +86,38 @@ class FakeDirectoryReader : public DirectoryReader {
       std::vector<std::string>();
 };
 
-class DirectoryReloaderCrlProviderTest : public ::testing::Test {
+class CrlProviderTest : public ::testing::Test {
  public:
   void SetUp() override {
+    std::string cert_string = GetFileContents(kRootCert.data());
+    X509* issuer = read_cert(cert_string);
+    crl_issuer = IssuerFromCert(issuer);
+    std::string intermediate_string =
+        GetFileContents(kCrlIntermediateIssuerPath.data());
+    X509* intermediate_issuer = read_cert(intermediate_string);
+    intermediate_crl_issuer = IssuerFromCert(intermediate_issuer);
+  }
+
+ protected:
+  std::string crl_issuer;
+  std::string intermediate_crl_issuer;
+  X509* read_cert(absl::string_view cert_string) {
```

Comment:
nit: Can we use C++ naming and call it "ReadCert" instead? Also would "ParsePemCert" be a more appropriate name?

Reply:
Done